microservices/notification/src/entities/message.ts

```diff
@@ -1,10 +1,16 @@
 import { IsTypeormDate, IsUndefinable } from '@lomray/microservice-helpers';
-import { Allow, IsEnum, IsObject, IsString, Length } from 'class-validator';
+import { Allow, IsArray, IsEnum, IsObject, IsString, Length } from 'class-validator';
 import { JSONSchema } from 'class-validator-jsonschema';
+import { Attachment } from 'nodemailer/lib/mailer';
 import { Entity, Column, CreateDateColumn, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
 import NotifyType from '@constants/notify-type';
 import type Notice from '@entities/notice';
 
+/**
+ * Attachment
+ */
+export interface IAttachment extends Pick<Attachment, 'filename' | 'encoding' | 'content'> {}
```

Comment:
move to separate file:
https://github.com/Lomray-Software/microservices/tree/dee8b143e5b9a54120cd53cc389a64ae764aae77/microservices/notification/src/interfaces/message-attachment